test/project.ts

```diff
@@ -405,7 +405,18 @@ describe("project integration tests", () => {
 
 				_.forEach(mobileHelper.platformNames, platform => {
 					let cordovaFile = util.format("cordova.%s.js", platform).toLowerCase();
-					assert.isTrue(fs.existsSync(path.join(projectDir, cordovaFile)), util.format("Cordova KendoUI.Drawer template does not contain mandatory '%s' file. This file is required in init command. You should check if this is problem with the template or change init command to use another file.", cordovaFile));
+					assert.isTrue(fs.existsSync(path.join(projectDir, cordovaFile)), `Cordova KendoUI.Drawer template does not contain mandatory '%s' file. This file is required in init command. You should check if this is problem with the template or change init command to use another file`);
```

Comment:
The message contains %s but I do not see an inserted string to replace it. Is this intended?